### PR TITLE
IMRPOVEMENT: Quotes use database now

### PR DESCRIPTION
**Pull request by Nanosplitter**
This PR makes quotes use a database instead of the `quotes.json` file.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 with open("config.yaml") as file:
     config = yaml.load(file, Loader=yaml.FullLoader)
 
-intents = nextcord.Intents.default()
+intents = nextcord.Intents.default().all()
```
Comment (Nanosplitter) [inline bot.py:17]: This gives the bot all the intents needed. Likely not needed for the main bot but makes making testing bots easier.

File: cogs/quotes.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import mysql.connector
+import json
+from noncommands import quotes
+
+
+with open("config.yaml") as file:
+    config = yaml.load(file, Loader=yaml.FullLoader)
+
+# Here we name the cog and create a new class for the cog.
+class Quotes(commands.Cog, name="quotes"):
```
Comment (Nanosplitter) [inline cogs/quotes.py:11]: Moved the quote-related commands to their own file

File: cogs/general.py [hunk 3]
```diff
@@ -112,28 +112,6 @@ async def ping(self, context):
...
-    @commands.command(name="newquote")
-    async def newquote(self, context):
-        """
-        [(Required) Quote] Creates a new quote to be put into the list of CS quotes.
-        """
-        quoteClass = quotes.Quotes(self.bot)
-        newquote=await quoteClass.newquote(context)
-
-        await context.send("Quote submitted! Quote: " + newquote)
-
```
Comment (Nanosplitter) [inline cogs/general.py:136]: Got moved to another file

File: noncommands/quotes.py [hunk 4]
```diff
@@ -2,52 +2,61 @@
...
+            host=config["dbhost"],
+            user=config["dbuser"],
+            password=config["dbpassword"],
+            database=config["databasename"],
+            autocommit=True,
+            use_unicode=True
+        )
+        mycursor = mydb.cursor(buffered=True)
+        sql = "SELECT quote FROM quotes WHERE quote LIKE %s ORDER BY RAND() LIMIT 1"
+        val = ("%" + keywords + "%",)
```
Comment (Nanosplitter) [inline noncommands/quotes.py:30]: This is automatically case-insensitive

**Review by nstringham — APPROVED** [review]
didn't look super closely but seams fine at first glance

**Pull request merged**